linux/device.go

```diff
@@ -621,7 +621,17 @@ func cleanupStaleScsiPaths(volume *model.Volume) (err error) {
 
 		// obtain current path serial using sysfs vpd80 page
 		var serialNumber string
-		serialNumber, err = getVpd80FromSysfs(h, c, t, l)
+		vendorName, err := getVendorFromSysfs(h, c, t, l)
+		if err != nil {
```

Comment:
We should ignore the error and only care about the vendorName so that other vendors can proceed irrespective of the error.
please make the change in all occurrences